docs/tables/azuread_authorization_policy.md

```diff
@@ -0,0 +1,29 @@
+# Table: azuread_application
+
+Represents a policy that can control Azure Active Directory authorization settings.
+
+## Examples
```

Comment:
@Subhajit97 Can you please add 2-3 more examples for this table? Is there additional info we'd want to pull out of the `default_user_role_permissions` or check against? Similarly, do any of the bool/string columns have info users would want to see easily?